Makefile

```diff
@@ -536,3 +536,40 @@ build-helm-docs:
 # Note: requires node and yarn to be installed
 build-ui:
 	cd $(ROOT_DIR)/sdk/python/feast/ui && yarn upgrade @feast-dev/feast-ui --latest && yarn install && npm run build --omit=dev
+
+
+
+# Go SDK & embedded
+install-protoc-dependencies:
+	pip install "protobuf>=4.24.0,<5.0.0" "grpcio-tools>=1.56.2,<2" "mypy-protobuf>=3.1"
+
+install-go-proto-dependencies:
+	go install google.golang.org/protobuf/cmd/protoc-gen-go@v1.31.0
+	go install google.golang.org/grpc/cmd/protoc-gen-go-grpc@v1.3.0
+
+install-go-ci-dependencies:
```

Comment:
this is all commented out, do we need it?

Reply:
commented out as I'm not sure if this setting is good/ready for CI test.